android/app/src/main/java/net/minetest/minetest/MainActivity.java

```diff
@@ -114,6 +126,30 @@ private void startNative() {
 		startActivity(intent);
 	}
 
+	@RequiresApi(Build.VERSION_CODES.O)
+	private void createNotificationChannel() {
+		NotificationManager notifyManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+		if (notifyManager == null)
+			return;
+
+		String channelName = getString(R.string.notification_channel_name);
+		String channelDescription = getString(R.string.notification_channel_description);
+		int importance = NotificationManager.IMPORTANCE_LOW;
+
+		NotificationChannel notifyChannel;
+		notifyChannel = notifyManager.getNotificationChannel(NOTIFICATION_CHANNEL_ID);
+		if (notifyChannel != null)
+			return;
```

Comment:
I think you can just safely always create the channel
this is also needed so it works correctly when the language changes.

Reply:
Oh, that is true. Fixed.